django/test/runner.py

```diff
@@ -585,6 +644,20 @@ def setup_test_environment(self, **kwargs):
         setup_test_environment(debug=self.debug_mode)
         unittest.installHandler()
 
+    def setup_shuffler(self):
+        if self.shuffle is False:
+            return
+        if self.shuffle is None:
+            shuffle_seed = random.randint(0, self.max_shuffle_seed)
+            shuffle_kind = 'generated'
+        else:
+            shuffle_seed = self.shuffle
+            shuffle_kind = 'given'
+        if self.verbosity >= 1:
+            print(f'Using shuffle seed: {shuffle_seed} ({shuffle_kind})')
+        self.shuffle_seed = shuffle_seed
```

Comment:
I don't see any need for this attribute.

Reply:
The purpose of this attribute is to provide programmatic access to the seed that `DiscoverRunner` generated randomly. This lets people do things like e.g. write it to a file, etc. if they want to re-run tests with the same seed.